__tests__/app.test.js

```diff
@@ -127,26 +127,34 @@ describe('GET requests', () => {
                 expect(body.articles).toBeSorted({ key: "created_at", descending: true })
             })
         });
-        test('200 - should accept a topic query and respond with ', () => {
+        test('200 - articles endpoint should accept a topic query, filtering by specified value', () => {
```

Comment:
lets make a new test suite for this rather than changing previous tests

Reply:
I think it looks like I'd changed a test but I'd just added them to my describe (GET /api/articles) endpoint. I've moveed them now into a new describe block.

Reply:
this one has been changed? when it should stay as /api/articles